cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,68 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_mod-docs-content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
```

Comment:
```suggestion
= Image scanning by by using the roxctl CLI
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

Reply:
I'm afraid that you might have overlooked the title. 
Current title: "Image scanning by using the roxctl CLI"
I don't think we need any more changes to the title.